gobblin-core/src/main/java/gobblin/source/extractor/hadoop/AvroFsHelper.java

```diff
@@ -165,16 +95,16 @@ public Schema getAvroSchema(String file) throws FileBasedHelperException {
 
   public DataFileReader<GenericRecord> getAvroFile(String file) throws FileBasedHelperException {
```

Comment:
Please add Javadoc for this method and make it clear that the caller is responsible for closing the returned `DataFileReader`.